adapter-guide/develop-stix-adapter.md

```diff
@@ -0,0 +1,602 @@
+
+* When converting from STIX, the data source query language requires specific data formats. For example, time stamps. In this case, the format of a value in the STIX pattern must be transformed during pattern translation if the STIX and query language data formats are different. 
+* When converting to STIX, the STIX object requires specific data formats. In this case, the format of a value that is returned in the data source results must be transformed during translation into a bundle of STIX objects. See [STIX™ Version 2.0. Part 4: Cyber Observable Objects](http://docs.oasis-open.org/cti/stix/v2.0/stix-v2.0-part4-cyber-observable-objects.html) for STIX data formats.
+
+
+#### Step 7. Update the MANIFEST.in file to include the path to the json mapping folder
+{: #translation-7}
+
+This file is required by Python so that the new mapping files can be found when STIX-shifter is packaged.
+
+#### Step 8. Verify that the translation module was created successfully
+{: #translation-8}
+
+You must have access to the data source either through a UI or CLI so that you can run the translated query and confirm that it works.
+The translation module can be tested by calling the `main.py file` from the command line and passing in the required arguments. The order of arguments is as follows:
+
+```
+python main.py translate <data source (module) name> <"query" or "result"> <STIX identity object> <pattern or JSON results to 
+be translated> <options>
+```
+
+##### Test the STIX pattern to data source query translation
+
+1.	Run the translation module from the command line. For example, using abc as a data source:
+
+```
+python main.py translate abc query '{}' "[network-traffic:src_port = 37020 and network-traffic:dst_port = 635] OR [ipv4-addr:value = '333.333.333.0'] AND [url:value = 'www.example.com'] START t'2019-01-28T12:24:01.009Z' STOP t'2019-01-28T12:54:01.009Z'"
+```
+
+2.	Visually verify the returned query by running it against the data source.
+
+##### Test the JSON data source results to STIX translation
+
+1.	Run the translation module from the command line. For example, using abc as a data source:
+
+```
+python main.py translate abc results '{"type": "identity","id": "identity--f431f809-377b-45e0-aa1c-
+6a4751cae5ff","name": "abc","identity_class": "events"}' '[{"Url": "www.example.com", "SourcePort": 3000, "DestinationPort": 1000, "SourceIpV4": "192.0.2.0", "DestinationIpV4": "198.51.100.0", "NetworkProtocol": "TCP"}]'
+```
+
+2.	Visually verify that all expected data is in the returned STIX bundle. If a data source field in your sample results is mapped in `to_stix_map.json`, the value must be in the STIX bundle under the mapped STIX property.
+
+**Note:** 
+* The <STIX identity object> represents a data source and is the first observed-data object that gets added to the STIX bundle during results translation. 
```

Comment:
I think you need to escape the arrow brackets with a back slash. Right now the "STIX identity object" isn't showing up. So it would be \\\< \\\> in the markdown code